common/source/docs/common-holybro-kakuteh7mini-v13.rst

```diff
@@ -1,7 +1,7 @@
 .. _common-holybro-kakuteh7mini-v13:
 
 ======================
```

Comment:
```suggestion
===========================
```
otherwise this causes a build error with title underlines not matching title length